js/app/cart/index.jsx

```diff
@@ -9,6 +9,8 @@ let isXsDevice = $('.visible-xs').is(':visible')
 var cart = document.getElementById('cart');
 var cartComponent;
 
+var sameTopCart = window.AppData.restaurantId === window.AppData.sessionRestaurantId;
```

Comment:
IMHO the variable name is misleading - it should be sthg like `isSessionCartForCurrentRestaurant`